src/Arblib.jl

```diff
@@ -41,6 +42,25 @@ macro libflint(function_name)
     return (:($function_name), Arb_jll.libflint)
 end
 
+const __isthreaded = Ref(false)
+
+function __init__()
+    __isthreaded[] = Base.get(ENV, "NEMO_THREADED", "") == "1"
+
+    if __isthreaded[]
+        ccall((:__gmp_set_memory_functions, :libgmp), Nothing,
+            (Int, Int, Int), 0, 0, 0)
+    end
+end
+
+function flint_set_num_threads(a::Int)
```

Comment:
```suggestion
function flint_set_num_threads(a::Integer)
```
anything that `cconverts` to `Clong` should be fine